test/e2e/storage/pd.go

```diff
@@ -405,7 +405,7 @@ var _ = utils.SIGDescribe("Pod Disks", func() {
 					framework.ExpectEqual(numNodes, origNodeCnt, fmt.Sprintf("Requires current node count (%d) to return to original node count (%d)", numNodes, origNodeCnt))
 					output, err = gceCloud.ListInstanceNames(framework.TestContext.CloudConfig.ProjectID, framework.TestContext.CloudConfig.Zone)
 					framework.ExpectNoError(err, fmt.Sprintf("Unable to get list of node instances err=%v output=%s", err, output))
-					gomega.Expect(false, strings.Contains(string(output), string(host0Name)))
+					framework.ExpectEqual(false, strings.Contains(string(output), string(host0Name)))
```

Comment:
lets be consistent throughout with "got before expect"

Reply:
@davidz627 Thanks,I will update it.

Reply:
some of the parameters put the "expected value" first, some put the "got value" first. For example this one is `false, strings.Contains(string(output), string(host0Name))` but there are other invocations with the ordering: `strings.Contains(string(output), string(host0Name)), false`. We should be consistent throughout on the ordering (which value we are getting from a call and which value is our expected value"

Reply:
@davidz627 I get the point,Thanks
you mean:
gomega.Expect(false, strings.Contains(string(output), string(host0Name)))
framework.ExpectEqual(false, strings.Contains(string(output), string(host0Name)))
==>
framework.ExpectEqual(strings.Contains(string(output), string(host0Name)),false)
/cc @oomichi 
i guess both are ok,but prefer framework.ExpectEqual(strings.Contains(string(output), string(host0Name)),false).What do you think?